front/src/js/list.js

```diff
@@ -1,8 +1,11 @@
 import { Toast } from "./toast.js"
-let toast
+import { EditTaskPopup } from "./edit-task.popup.js"
+
+let toast, editTaskPopup
 
 document.addEventListener("DOMContentLoaded", function() {
     toast = new Toast()
+    editTaskPopup = new EditTaskPopup()
```

Comment:
Since the popup is inserted into DOM as soon as the page is loaded, perhaps the core structure should be in the HTML? Or it should be inserted later?